source/core/UnitTest.cpp

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
I did, and I think it's something we could fairly easily migrate too, but it seemed overkill for a handful of tests, and more work to incorporate into the build for all platforms.